src/sql/magic.py

```diff
@@ -640,6 +674,38 @@ def _persist_dataframe(
         )
 
 
+def get_query_type(command: str):
+    """
+    Returns the query type of the original sql command
+    """
+    query_type = (
+        sqlparse.parse(command)[0].get_type() if sqlparse.parse(command) else None
+    )
+    if query_type == "UNKNOWN":
+        return None
+    return query_type
+
+
+def get_dependency_in_CTE(sql_original, dependencies):
```

Comment:
do not use capital letters in function names

Reply:
This function is no longer in use as I've removed it and provided a more generic warning.

Reply:
[Fixed](https://github.com/ploomber/jupysql/pull/878/commits/a204e262aa93b6bf06494dd2e06c1f27b25e0878)